src/draco/attributes/geometry_attribute.h

```diff
@@ -112,28 +112,40 @@ class GeometryAttribute {
     if (out_val == nullptr)
       return false;
     switch (data_type_) {
-      case DT_FLOAT32:
-        return ConvertTypedValue<float, OutT, out_att_components_t>(att_id,
-                                                                    out_val);
-      case DT_UINT8:
-        return ConvertTypedValue<uint8_t, OutT, out_att_components_t>(att_id,
-                                                                      out_val);
-      case DT_INT8:
+	  case DT_INT8:
         return ConvertTypedValue<int8_t, OutT, out_att_components_t>(att_id,
                                                                      out_val);
-      case DT_UINT16:
-        return ConvertTypedValue<uint16_t, OutT, out_att_components_t>(att_id,
-                                                                       out_val);
+	  case DT_UINT8:
+		  return ConvertTypedValue<uint8_t, OutT, out_att_components_t>(att_id,
```

Comment:
same  for the `return`,  every new block should start two spaces from the previous one